athena-neptune/src/main/resources/log4j2.xml

```diff
@@ -29,7 +29,8 @@
     </Appenders>
     <Loggers>
         <Logger name="com.amazonaws.athena.connector.lambda" level="${env:ATHENA_FEDERATION_SDK_LOG_LEVEL:-warn}" />
-        <Logger name="com.amazonaws.athena.connectors.neptune" level="${env:ATHENA_FEDERATION_SDK_LOG_LEVEL:-warn}" />
+        <Logger name="com.amazonaws.athena.connectors.neptune" level="warn" />
```

Comment:
please revert the changes and keep as: `level="${env:ATHENA_FEDERATION_SDK_LOG_LEVEL:-warn}`

This is used for customer debugging so that log levels can be configured via lambda env variable

Reply:
changed